test/Worker.Extensions.Tests/Worker.Extensions.Tests.csproj

```diff
@@ -19,6 +19,13 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
+        <PackageReference Include="SmartAnalyzers.RoslynTestKit" Version="4.7.0" />
+        <PackageReference Include="coverlet.collector" Version="3.2.0">
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
The changes here are taken from [Sdk.Analyzers.Tests.csproj](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/test/Sdk.Analyzers.Tests/Sdk.Analyzers.Tests.csproj) as we wanted to keep _extension analyzer tests_ outside of `Sdk.Analyzer.Tests`.